test_python.py

```diff
@@ -1,13 +1,153 @@
 import unittest
+from pathlib import Path
 
-from needletail import parse_fastx_file, parse_fastx_string, NeedletailError, reverse_complement, normalize_seq
-
+from needletail import (
+    NeedletailError,
+    Record,
+    normalize_seq,
+    parse_fastx_file,
+    parse_fastx_string,
+    reverse_complement,
+)
 
 FASTA_FILE = "./tests/data/test.fa"
 FASTQ_FILE = "./tests/specimen/FASTQ/example.fastq"
 
 
-class ParsingTestCase(unittest.TestCase):
+class RecordClassTestCase(unittest.TestCase):
```

Comment:
can you add tests for `str(record)` @apcamargo ?

Reply:
I think those are already there. In lines 63-68:

```py
    def test_record_str(self):
        self.assertEqual(str(Record("test", "AGCTGATCGA")), ">test\nAGCTGATCGA\n")
        self.assertEqual(
            str(Record("test", "AGCTGATCGA", ";**9;;????")),
            "@test\nAGCTGATCGA\n+\n;**9;;????\n",
        )
```